sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/text/TextSuite.scala

```diff
@@ -33,8 +33,8 @@ class TextSuite extends QueryTest with SharedSQLContext {
     verifyFrame(sqlContext.read.text(testFile))
   }
 
-  test("writing") {
-    val df = sqlContext.read.text(testFile)
+  test("SPARK-12562 verify write.text() can handle column name beyond `value`") {
+    val df = sqlContext.read.text(testFile).withColumnRenamed("value", "adwrasdf")
```

Comment:
This might be failed because the later method `verifyFrame` will check if the df read in has schema like `new StructType().add("value", StringType)`. You could update `verifyFrame` to check if it has only one StringType column.